app/views/tag/_tags_popover.html.erb

```diff
@@ -3,7 +3,11 @@
 	<%= form_with remote: true, url: '/subscribe/multiple/tag/:tagnames' do |f| %>
 		<% tags.each do |tag| %>
 			<div>
-				<input type="checkbox" id="<%= tag.id %>" name="tagnames[]" value="<%= tag.name %>"><span class="label <%= label_name %>" style="margin-left: 3px; margin-right: 10px; display: inline-block;"> <%= tag.name %> </span>
+				<% if current_user.following(tag.name) %>
```

Comment:
There are indentation conventions in RoR. I guess we use 2 space indentations. Please correct them in each file.